mw-webapp/src/converter/MonthReportConverter.ts

```diff
@@ -0,0 +1,50 @@
+import {MonthReport} from "src/model/businessModel/MonthReport";
+import {JobDone} from "src/model/businessModel/JobDone";
+import {PlanForNextPeriod} from "src/model/businessModel/PlanForNextPeriod";
+import {CurrentProblem} from "src/model/businessModel/CurrentProblem";
+import {MonthReport as MonthReportDTO} from "src/model/firebaseCollection/MonthReport";
+import {JobDoneService} from "src/service/JobDoneService";
+import {PlanForNextPeriodService} from "src/service/PlanForNextPeriodService";
+import {CurrentProblemService} from "src/service/CurrentProblemService";
+import {MentorCommentService} from "src/service/MentorCommentService";
+import {MentorComment} from "src/model/businessModel/MentorComment";
+
+const jobsDoneRaw = await JobDoneService.getValueFromRealTimeDb();
+const plansForNextPeriodRaw = await PlanForNextPeriodService.getValueFromRealTimeDb();
+const problemsForCurrentPeriodRaw = await CurrentProblemService.getValueFromRealTimeDb();
+const mentorCommentsRaw = await MentorCommentService.getValueFromRealTimeDb();
+
+export const MonthReportDTOToMonthReportConverter = (monthReportRaw: MonthReportDTO) => {
+  const jobsDone = monthReportRaw.jobsDone?.map((item) => {
+    const jobDone: JobDone = jobsDoneRaw
+      .find((elem) => elem.uuid === item) || jobsDoneRaw[0];
+    return jobDone;
+  });
+
+  const plansForNextPeriod = monthReportRaw.plansForNextPeriod?.map((item) => {
```

Comment:
looks like you can avoid duplicates by using function with just two args in this case